<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BSM Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="aOpen"/>
    <addaction name="aReload"/>
    <addaction name="separator"/>
    <addaction name="aSave"/>
    <addaction name="aSaveAs"/>
    <addaction name="separator"/>
    <addaction name="aClose"/>
    <addaction name="separator"/>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="aAddLanguage"/>
    <addaction name="aRemoveLanguage"/>
    <addaction name="separator"/>
    <addaction name="aAddTranslation"/>
    <addaction name="aRemoveTranslation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="tbFile">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aOpen"/>
   <addaction name="aSave"/>
   <addaction name="aSaveAs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="tbEdit">
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aAddLanguage"/>
   <addaction name="aRemoveLanguage"/>
   <addaction name="separator"/>
   <addaction name="aAddTranslation"/>
   <addaction name="aRemoveTranslation"/>
  </widget>
  <action name="aOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/document-open-5.ico</normaloff>:/32x32/actions/document-open-5.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="aExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/application-exit-4.ico</normaloff>:/32x32/actions/application-exit-4.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Close program</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="aAddLanguage">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/language-add.ico</normaloff>:/32x32/actions/language-add.ico</iconset>
   </property>
   <property name="text">
    <string>Add language</string>
   </property>
   <property name="toolTip">
    <string>Add language</string>
   </property>
  </action>
  <action name="aReload">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/document-open-recent.ico</normaloff>:/32x32/actions/document-open-recent.ico</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload file from disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="aClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/document-close-4.ico</normaloff>:/32x32/actions/document-close-4.ico</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close file</string>
   </property>
  </action>
  <action name="aSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/document-save-5.ico</normaloff>:/32x32/actions/document-save-5.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="aSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/document-save-as-5.ico</normaloff>:/32x32/actions/document-save-as-5.ico</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save file as...</string>
   </property>
  </action>
  <action name="aRemoveLanguage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/language-remove.ico</normaloff>:/32x32/actions/language-remove.ico</iconset>
   </property>
   <property name="text">
    <string>Remove language</string>
   </property>
  </action>
  <action name="aAddTranslation">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/list-add-4.ico</normaloff>:/32x32/actions/list-add-4.ico</iconset>
   </property>
   <property name="text">
    <string>Add translation</string>
   </property>
   <property name="toolTip">
    <string>Add translation</string>
   </property>
  </action>
  <action name="aRemoveTranslation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/32x32/actions/list-remove-4.ico</normaloff>:/32x32/actions/list-remove-4.ico</iconset>
   </property>
   <property name="text">
    <string>Remove translation</string>
   </property>
   <property name="toolTip">
    <string>Remove translation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>aExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
